modules/neovim/init/spellcheck.nix

```diff
@@ -25,9 +28,45 @@ in {
         A list of languages that should be used for spellchecking.
 
         To add your own language files, you may place your `spell`
-        directory in either `~/.config/nvim` or the
+        directory in either {file}`~/.config/nvim` or the
         [additionalRuntimePaths](#opt-vim.additionalRuntimePaths)
-        directory provided by **nvf**.
+        directory provided by nvf.
+      '';
+    };
+
+    extraSpellFiles = mkOption {
```

Comment:
The name is a little misleading, I wouldn't include "file" in it

Reply:
What's a better name? I was considering `extraSpellWords` but that's also misleading since we expect a key-value pair of `name = list`

Reply:
extraSpellWords is fine, maybe extraSpellLists/extraSpellWordLists, although the last one is too long

Reply:
I think `extraSpellWords` is the best approach here. `extrasSpellLists` is equally misleading, and the latter is simply too long. The name gives you a hint to its purpose, and the documentation shows you the usage.